src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,148 @@
+- Clusters with Enterprise Free or Enterprise Trial licenses cannot disable telemetry; if such a cluster signals a telemetry sending error, it will be due to firewall configuration or a network issue.
+{{site.data.alerts.end}}
+
+## Obtain a license
+
+To obtain a paid **Enterprise** license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a>.
+
+To obtain an **Enterprise Free** or **Enterprise Trial** license, take the following steps:
+
+1. Point your web browser to the [CockroachDB {{ site.data.products.cloud }} Console](https://cockroachlabs.cloud) and [log in]({% link cockroachcloud/authentication.md  %}#console-ui-authentication).
+1. Select **Organization &raquo; Enterprise Licenses** from the nav bar dropdown. This will bring you to the **Enterprise Licenses** page, which shows a (possibly empty) list of licenses and their keys, with information about the status of each.
+1. Click the **Create License** button. This will bring you to a page called **Get started with CockroachDB Enterprise**.
+1. On this page, you will create an **Enterprise Trial** license or an **Enterprise Free** license.
+    1. To create an **Enterprise Trial** license:
+        1. Fill in the form with the required information.
+        1. Click the **Continue** button.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+    1. To create an **Enterprise Free** license:
+        1. Fill in the form with the required information.
+        1. Toggle the switch called **Find out if my company qualifies for an Enterprise Free license**. *By toggling this switch you are legally attesting to the fact that your company revenue meets the requirements of the license*.
+        1. Click the **Continue** button.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+
+{{site.data.alerts.callout_danger}}
+You will not be able to create more than one **Enterprise Trial** license per day. If you try, the UI will prevent you from proceeding, and the following message will be displayed:
+
+> A new Enterprise Trial license cannot be created at this time because one was recently created. Tell us more about your intended use to find out if you qualify for an Enterprise Free license, or create a new one after October 16, 2024 at 10:49 AM EDT.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+For quick local testing of Enterprise features, you can [run a single-node cluster](#single-node-clusters).
+{{site.data.alerts.end}}
+
+## Set a license
+
+{% include common/set-enterprise-license.md %}
```

Comment:
Should we add a disclaimer here that you cannot apply a Enterprise Trial license more than once to a cluster?

Reply:
I added a note to the bottom of the included instructions that 

"You cannot apply an **Enterprise Trial** license more than once to the same cluster."

figured i'd keep it short b/c we document elsewhere the UI message re: not giving out multiple Trial licenses, pls contact Sales etc - let me know if that works!